src/Opdex.Platform.Domain/Models/IndexLock.cs

```diff
@@ -22,5 +22,5 @@ public IndexLock(bool available, bool locked, string instanceId, IndexLockReason
 
 public enum IndexLockReason
 {
-    Rewind = 1, Index = 2, Deploy = 3
+    Deploying = 1, Indexing = 2, Rewinding = 3
```

Comment:
Not to myself, will need to update client for maintenance notifications with this pr